chain/client/src/client_actor.rs

```diff
@@ -983,6 +983,9 @@ impl ClientActor {
 
             let last_final_hash = *block.header().last_final_block();
 
+            for chunk in block.chunks().iter() {
```

Comment:
Please check if the chunk is included.

Reply:
Hopefully done, if I understood correctly how chunk masks work